### PR TITLE
Remove double spacing in text area

### PR DESCRIPTION
**Pull request by emmaviolet**
Also adds a textarea placeholder to sandbox for future use.

PT: https://www.pivotaltracker.com/story/show/93029542

Devs: @emmaviolet @igoratron

### CODE REVIEW

**Comment by emmaviolet** [comment]
Ping @FundingCircle/navy-seal @RazvanDH

File: src/sass/general/forms-v2.scss [hunk 1]
```diff
@@ -181,7 +181,7 @@
   input:not([type="checkbox"]):not([type="radio"]):not([type="button"]):not([type="submit"]),
   .select,
   textarea {
-    @include font(0, 2.5);
+    @include font(0, false);
```
Comment (RazvanDH) [inline src/sass/general/forms-v2.scss]: Why did you remove the line-height?

Comment (emmaviolet) [inline src/sass/general/forms-v2.scss]: Because it was double-spacing it (see the PT ticket)

**Comment by RazvanDH** [comment]
Can you please remove the `dist` files commited? Those will be added when we rebuild and tag master after merge.

**Comment by emmaviolet** [comment]
@RazvanDH yep, :recycle:

**Comment by emmaviolet** [comment]
@RazvanDH this is all :recycle: as per your thoughts and good to go - thank you!

**Comment by RazvanDH** [comment]
:+1: LGTM
